deepmd/pt/model/task/fitting.py

```
@@ -459,29 +464,30 @@ def _forward_common(
-            t_aparam_avg = self._extend_a_avg_std(self.aparam_avg, nb, nloc)
-            t_aparam_inv_std = self._extend_a_avg_std(self.aparam_inv_std, nb, nloc)
-            aparam = (aparam - t_aparam_avg) * t_aparam_inv_std
-            xx = torch.cat(
-                [xx, aparam],
-                dim=-1,
-            )
-            if xx_zeros is not None:
-                xx_zeros = torch.cat(
-                    [xx_zeros, aparam],
+        if not self.use_aparam_as_mask:
+            if self.numb_aparam > 0:
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)